selenium_webdriver/src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,28 @@
+package tests;
+
+import core.driver.DriverManagerFactory;
+import core.enums.DriverType;
+import core.service.TestDataReader;
+import core.utils.TestListener;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Listeners;
+
+@Listeners({TestListener.class})
+public class BaseTest {
+
+    protected WebDriver driver;
```

Comment:
Why do you need to WebDriver instance here?

Reply:
Fixed